cockpit/leapp.html

```diff
@@ -139,7 +140,18 @@
             list_local_vms();
         });
 
+        function set_lock_cmd() {
+            $("input[name=source-machine], input[name=target-machine]").attr("disabled", true);
+            $("#migrate-button").attr("disabled", true);
```

Comment:
one could also use a new class 'ui-lockable' and use that as indicator, I guess that's easier, no?